### PR TITLE
Support cancellation

### PR DESCRIPTION
**Pull request by andreasohlund**

### CODE REVIEW

File: src/NServiceBus.Callbacks/NServiceBus.Callbacks.csproj [hunk 1]
```diff
@@ -9,7 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="NServiceBus" Version="[8.0.0-alpha.630, 9.0.0)" />
+    <PackageReference Include="NServiceBus" Version="8.0.0-alpha.852" />
```
Comment (andreasohlund) [inline src/NServiceBus.Callbacks/NServiceBus.Callbacks.csproj:12]: Were we supposed to keep the ranges?

Comment (DavidBoike) [reply src/NServiceBus.Callbacks/NServiceBus.Callbacks.csproj:12]: Yes, packageable components are supposed to have the ranges. Testing packages are supposed to be just the latest version.

File: src/NServiceBus.Callbacks.Testing/NServiceBus.Callbacks.Testing.csproj [hunk 2]
```diff
@@ -14,7 +14,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="NServiceBus.Testing" Version="[8.0.0-alpha.98, 9.0.0)" />
+    <PackageReference Include="NServiceBus" Version="8.0.0-alpha.852" />
+    <PackageReference Include="NServiceBus.Testing" Version="8.0.0-alpha.152" />
```
Comment (DavidBoike) [inline src/NServiceBus.Callbacks.Testing/NServiceBus.Callbacks.Testing.csproj:18]: Probably shouldn't add the transitive reference? In this case though, I'm really confused why the reference to the Testing package was a range.

**Review by DavidBoike — APPROVED** [review]
Just a couple changes with packages, code is fine.

File: src/NServiceBus.Callbacks/NServiceBus.Callbacks.csproj (hunk 1 above)
Comment (andreasohlund) [reply src/NServiceBus.Callbacks/NServiceBus.Callbacks.csproj]: fixed

**Comment by andreasohlund** [comment]
This one is ready for final review

**Review by adamralph — APPROVED** [review]

**Pull request merged**